sdk/security/keyvault/azadmin/backup/client_test.go

```diff
@@ -21,13 +22,23 @@ import (
 
 func TestBackupRestore(t *testing.T) {
 	client, sasToken := startBackupTest(t)
-
 	testSerde(t, &sasToken)
 
+	// if in live mode, wait 30 seconds before polling again
+	// in playback mode, we don't need that delay, so shortening the retry frequency
+	var frequency time.Duration
+	if recording.GetRecordMode() != recording.PlaybackMode {
+		frequency = time.Second * 30
+	} else {
+		frequency = time.Second
+	}
```

Comment:
Can we simply pass a zero `Duration` when not in playback and thereby accept the default?
```suggestion
	var frequency time.Duration
	if recording.GetRecordMode() == recording.PlaybackMode {
		frequency = time.Second
	}
```

Reply:
Good idea, thanks!